agent/engine/docker_task_engine.go

```diff
@@ -109,6 +109,8 @@ const (
 	stopContainerBackoffJitter     = 0.2
 	stopContainerBackoffMultiplier = 1.3
 	stopContainerMaxRetryCount     = 5
+
+	serviceConnectAppnetAgenTarballDir = "/var/lib/ecs/deps/serviceconnect/"
```

Comment:
It just occurred to me that the `/var/lib/ecs/deps/` is the host directory, and is mounted into container as `/managed/agents/` (see the bind mount configuration [here](https://github.com/aws/amazon-ecs-agent/blob/f003b0fed6b8e12daec448cdef34b37757c297a1/ecs-init/docker/docker.go#L121-L122))

I thought the watcher should be created on the container path (because the exact path does not exist in Agent container technically), but are we observing expected behaviors even when the watcher is created with the host path?

Reply:
You are right, watching container path is appropriate here, made changes in the latest rev accordingly.